DB/migration/pending/3.sql

```diff
@@ -0,0 +1,46 @@
+create type "public"."brief_response_answer_question_enum" as enum ('essentialRequirements', 'niceToHaveRequirements', 'availability', 'dayRate', 'specialistName', 'attachedDocumentURL');
```

Comment:
make this `1.sql` and remove the existing files - they are both already in prod

Reply:
fixed in latest commit